tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/model/ToneChatOptions.java

```diff
@@ -24,7 +24,8 @@
 public class ToneChatOptions extends GenericModel {
 
   /**
-   * The desired language of the response. For two-character arguments, regional variants are treated as their parent language; for example, `en-US` is interpreted as `en`.
+   * The desired language of the response. For two-character arguments, regional variants are treated as their parent
+   * language; for example, `en-US` is interpreted as `en`.
```

Comment:
The formatting changes in these files are crazy-making!  We have to get to a point where the formatter is _*stable*_, or it will not be possible to do quality reviews.